buildbot/master.cfg

```diff
@@ -549,6 +612,86 @@ def make_dolphin_linux_build(mode="normal"):
                                       hideStepIf=StepWasSuccessful))
     return f
 
+def make_dolphin_linux_steamrt_build(mode="normal"):
+    f = BuildFactory()
+
+    mode = mode.split(",")
+
+    normal = "normal" in mode
+    pr = "pr" in mode
+
+    f.addStep(GitNoBranch(repourl="https://github.com/dolphin-emu/dolphin.git",
+                          progress=True, mode="incremental", workdir="build/qt"))
```

Comment:
That workdir seems wrong.

Reply:
Would `build/build/qt` work? Was worried putting it into the CMake build directory would mess something up.